nengo/config.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, configures):
         assert inspect.isclass(configures)
         self._configures = configures
         self._extraparams = {}
-        self._default_params = tuple(
+        self._defaultparams = tuple(
```

Comment:
I prefer them with, and that's also what the API is like right now. Easier to change the internals to conform with that.